pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,332 @@
+---
+$title: Use AMP components in non-AMP pages
+$order: 1
+description: 'Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP.'
+tutorial: false
+toc: true
+formats:
+  - websites
+author: CrystalOnScript
+---
+
+Use AMP's well-tested, cross-browser compatible and accessible components anywhere on the web with Bento AMP. Bento AMP allows you to use AMP components in non-AMP pages without needing to commit to fully valid AMP! Bento allows you to take components and place them in implementations with frameworks and CMSs that don't support AMP. 
```

Comment:
Not sure about this, but can we avoid starting two subsequent sentences with the same "Bento allows you to ..."?